utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/SchemaBasedParameter.scala

```diff
@@ -28,7 +29,7 @@ sealed trait SchemaBasedParameter {
 
 object SchemaBasedParameter {
   type RecordFieldName = String
-  type ParameterName   = String
+  type ParameterName   = pl.touk.nussknacker.engine.api.parameter.ParameterName
```

Comment:
```suggestion
  type ParameterName   = ParameterName
```

Reply:
I removed the type def. It's not needed now.